index.html

```diff
@@ -67,9 +67,47 @@ <h1 class="font-special light-rose">with <span class="font-special2 orange">US!<
 </div>
 
 <div class="functionality">
-    <div class="more"></div>
+    <header class="functionality-header my-4">
+        <h1 class="display-3">Z naszą aplikacją:</h1>
+    </header>
+    <div class="row text-center">
+        <div class="col-lg-3 col-md-12 col-sm-12 mb-4">
```

Comment:
nie powinno być od sm--->lg rosnąco ?

Reply:
hmm... tez mysle ze powinno byc chyba sm--->lg ale ze wzgledu na sama czytelnosc bo bootstrap jest mobile first. innymi slowy jezeli na przyklad piszesz "col-sm-6 col-lg-4" to znaczy ze bedzie col-md-6 pomiedzy bez pisanie tego bo od col-sm-6 do col-lg-4 nic nie dzieje czyli wartosc dla col-md tez powinna byc taka jak dla sm. wydaje mi sie ze kolejnosc pisania lg > md > sm czy sm > md > lg nie ma tak w zasadzie wiekszego znaczenia bo to po prostu wymienianie kolejnych klas ale zmienilbym to dla czytelnosci ;) mam nadzieje ze za bardzo nie namieszalem ;p